src/components/twilio-voice-dialer/public/index.html

```diff
@@ -1 +1,18 @@
-<!-- TODO: Implement example usage of the Web Component -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Twilio Voice Dialer</title>
+  </head>
+  <body>
+    <h1>Twilio Voice Dialer</h1>
+
+    <twilio-voice-dialer recipient=""></twilio-voice-dialer>
+
+    <script src="twilio.js"></script>
```

Comment:
Please add a todo and submit a jira to think about where to put this so we don't forget.